machineconfiguration/v1/0000_80_containerruntimeconfig.crd.yaml

```diff
@@ -113,10 +113,6 @@ spec:
                   items:
                     description: ContainerRuntimeConfigCondition defines the state of the ContainerRuntimeConfig
                     type: object
-                    required:
-                      - lastTransitionTime
-                      - status
-                      - type
```

Comment:
No longer required?

Reply:
Unforunately yes, because in the MCO originally it wasn't: 

https://github.com/openshift/machine-config-operator/blob/886c5c3fc7a96e922a65d83ecd39d19f4921dd50/install/0000_80_machine-config-operator_01_containerruntimeconfig.crd.yaml#L145

Reply:
Ahh right, so in the copy/paste some generation has made it required, and this reverts that? Would be good to squash that change somehow so it's obvious?

Perhaps if we do the copy/paste, then the next commit fixes all the optional vs required, then generate as the final commit, it will be more obvious that there's actually no diff?

Reply:
Thanks, I tried to squish the commits down to a more clear: 
-  add existing -> modify annotations -> generate

hopefully that's...at least a little better?

Reply:
That's what I was hoping for, let me re-review and i'll let you know if anything else can be done :)

Reply:
Ok so these required to optional transitions still exist even when the update comes at the end? Why are these fields switching from required to optional?

Reply:
Can you link to where this CRD currently gets installed into payload to show that these fields aren't correct there? If that's the case, at least we have a record of it then

Reply:
Ahh I see, you've done that already nvm